docs/reference/commandline/volume_create.md

```diff
@@ -0,0 +1,41 @@
+<!--[metadata]>
++++
+title = "volume create"
+description = "The volume create command description and usage"
+keywords = ["volume, create"]
+[menu.main]
+parent = "smn_cli"
+weight=1
++++
+<![end-metadata]-->
+
+# volume create
+
+  Usage: docker volume create [OPTIONS]
+
+  Create a volume
+
+    -d, --driver=local    Specify volume driver name
+    --help=false          Print usage
+    --name=               Sepcify volume name
+    -o, --opt=map[]       Set driver specific options
+
+The `docker volume create` command creates a new volume that can be consumed by
+a container to store data in. If a name is not specified, a random ID will be
+generated. A container can be configured to use a created volume:
+
+  $ docker volume create --name hello
```

Comment:
Not a docs thing, but we talked about this on IRC; seeing the above, `docker volume create` doesn't return the name of the volume that was created, so this isn't possible;

```bash
docker run -d -v $(docker volume create --driver=foo):/something busybox ls /something
```

It would be a shame if we didn't support that (either creating the volume in a subshell, or to store the volume name in a variable)